### DISCUSSION TITLE
"WALs waiting to be archived" in prometheus output?

### DISCUSSION THREAD

**Discussion by cleboo** [cloudnative-pg/cloudnative-pg#1831] [Q&A] [answered]
Hi!

Is the "WALs waiting to be archived" figure from `cnpg status` included in the prometheus output somewhere? Would be useful for monitoring backup status.

**Comment by gbartolini** [comment, marked as answer]
Please refer to the `cnpg_collector_pg_wal_archive_status{value="ready"}` metric as outlined in https://cloudnative-pg.io/documentation/current/monitoring/#predefined-set-of-metrics
